code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -185,6 +187,13 @@
 	if(slot == ITEM_SLOT_OCLOTHING) //we only give the mob the ability to toggle the helmet if he's wearing the hardsuit.
 		return 1
 
+/// Burn the person inside the hard suit just a little, the suit got really hot for a moment
```

Comment:
This should have some sort of feedback message so people know why they're suddenly getting roasted

Reply:
I added a notice to the wearer, and a chance to scream if you are burned.